Swabbr.Core/Helpers/SecureHashHelper.cs

```diff
@@ -0,0 +1,79 @@
+﻿using Microsoft.AspNetCore.Cryptography.KeyDerivation;
+using System;
+using System.Linq;
+using System.Security.Cryptography;
+
+namespace Swabbr.Core.Helpers
+{
+    /// <summary>
+    ///     Used to hash confidential information.
+    /// </summary>
+    /// <remarks>
+    ///     This was based on the Microsoft documentation for hashing passwords and
+    ///     depends on the KeyDerivation package which is standalone. The way we
+    ///     store our data and validate our hashes is based on the Identity Framework
+    ///     Password Hasher.
+    ///     https://docs.microsoft.com/en-us/aspnet/core/security/data-protection/consumer-apis/password-hashing?view=aspnetcore-5.0
+    ///     https://github.com/dotnet/AspNetCore/blob/main/src/Identity/Extensions.Core/src/PasswordHasher.cs
+    /// </remarks>
+    public static class SecureHashHelper
+    {
+        private const int SaltLengthInBits = 128;
+        private const int HashLengthInBits = 256;
+        private const int IterationCount = 10000;
```

Comment:
Consider using the digit separator `_` for long digits to improve readability.

```suggestion
        private const int IterationCount = 10_000;
```